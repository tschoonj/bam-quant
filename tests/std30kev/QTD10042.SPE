$DATA:
0 2000
15810
10445
4747
1568
438
100
40
23
15
5
1
7
2
0
7
20
245
2092
2753
2618
2623
2403
2384
2312
2303
2201
2131
2129
2087
2005
1949
2041
1897
1913
1946
1971
1955
1877
1963
1911
1934
1972
1957
2009
1955
1969
1866
1925
1971
1918
1903
1901
1861
1803
1798
1723
1655
1636
1555
1520
1480
1387
1321
1335
1189
1252
1231
1179
1166
1090
1105
1043
1118
1083
974
1021
872
896
819
842
830
758
720
729
777
726
688
683
722
680
683
693
686
652
651
615
656
663
626
646
670
671
686
638
626
599
629
617
572
531
560
544
580
535
515
547
513
455
540
440
489
506
450
479
487
443
452
466
430
446
494
464
481
488
439
441
423
472
486
421
412
425
422
456
425
424
459
457
409
438
408
417
391
438
386
412
435
395
452
362
436
377
415
444
443
375
377
393
386
401
378
394
367
384
382
374
395
369
392
359
383
382
392
389
350
374
352
353
406
371
335
365
388
349
363
380
337
376
368
338
365
408
346
335
349
347
337
350
341
346
348
347
343
363
364
341
325
344
350
313
338
366
379
328
367
334
330
341
332
353
337
349
337
346
350
313
330
318
311
325
355
335
274
348
356
329
343
316
322
322
326
284
310
292
308
322
341
320
291
321
293
329
328
330
344
362
345
358
352
400
337
341
342
300
305
314
340
339
292
320
301
302
296
291
307
287
301
307
300
284
308
325
317
315
300
300
328
309
290
310
286
272
307
296
303
262
286
294
306
339
316
345
355
392
431
499
556
548
541
479
450
403
353
333
316
298
297
272
276
309
271
266
285
282
282
308
297
284
277
275
270
286
303
296
282
301
321
339
322
318
339
308
324
320
314
300
296
269
261
258
278
258
297
288
245
296
280
317
304
315
324
298
280
278
293
252
284
274
273
312
257
259
267
251
273
264
270
283
296
313
269
308
292
277
279
297
299
309
274
293
296
299
275
261
257
273
277
298
269
260
311
290
276
298
239
296
279
274
286
245
264
260
278
275
264
304
274
304
316
280
290
301
296
254
275
290
286
257
258
288
280
284
322
263
267
262
305
255
270
285
287
308
263
292
289
305
293
279
311
321
306
297
277
242
278
255
234
262
287
256
267
304
273
281
301
287
278
254
280
252
264
260
276
261
271
278
255
286
281
272
263
270
283
256
257
289
285
278
298
270
283
283
276
247
305
260
243
274
257
289
271
272
303
267
289
243
286
271
293
266
306
272
284
285
290
285
286
294
272
256
289
266
293
265
271
290
282
273
287
273
281
250
291
303
288
267
285
270
280
298
294
309
246
248
281
283
274
300
295
279
288
286
277
278
276
309
263
271
302
298
306
301
326
297
295
307
312
270
247
279
346
296
310
298
297
281
319
311
329
308
304
292
303
275
244
270
275
305
337
291
297
301
301
270
300
355
289
342
326
331
299
283
309
305
314
320
324
315
327
317
295
347
374
328
318
299
334
320
364
306
314
310
335
311
323
304
325
323
316
330
345
329
373
325
295
376
343
341
377
346
364
377
299
347
325
348
331
350
341
334
348
355
347
345
337
360
360
355
375
344
328
373
372
391
362
379
398
404
388
371
393
388
375
399
367
389
379
396
397
371
398
400
401
408
396
415
418
396
432
426
447
456
441
433
447
399
392
461
416
435
412
481
452
430
465
426
442
485
420
474
452
467
484
512
487
447
479
467
489
480
476
504
489
509
516
506
496
533
513
560
504
511
547
518
539
538
504
508
532
534
561
569
530
536
530
554
604
582
577
590
614
611
640
667
677
733
730
784
807
969
1075
1287
1405
1675
1942
2223
2578
2664
2944
2997
2948
2903
2803
2626
2281
1965
1732
1609
1399
1241
1109
1074
1010
1046
970
1068
1082
1097
1133
1155
1179
1223
1281
1353
1430
1342
1451
1560
1527
1618
1643
1668
1708
1774
1821
1817
1885
1918
1928
2021
1996
2059
1960
2070
2166
2105
2113
2208
2210
2263
2320
2430
2453
2485
2547
2580
2709
2753
2880
2942
3059
3305
3500
3676
4157
4825
5837
7312
10023
14251
20835
30242
44356
63346
87589
119008
155370
197281
243500
290367
334101
375514
406600
426856
433314
425385
402820
368138
322507
271339
219416
172181
128749
92331
63806
42775
27847
17614
11434
7305
4853
3543
2684
2108
1931
1695
1554
1362
1317
1214
1147
1095
1064
997
978
939
921
904
915
898
809
802
777
830
858
829
801
753
820
796
857
778
761
835
807
821
835
805
825
833
777
867
847
813
835
797
865
897
903
867
884
893
913
951
989
938
1028
995
990
1055
1030
1095
1093
1133
1267
1320
1207
1281
1348
1418
1418
1553
1588
1723
1919
2254
2802
3697
5081
7194
10145
14408
20071
27585
35730
45694
56314
66296
75041
80271
84005
83563
80832
74258
66045
56386
46618
36786
28769
22154
17326
14318
12098
11499
11597
11788
12853
13122
13419
13111
12617
11412
10069
8241
6891
5511
4110
2823
2023
1369
874
576
348
232
158
119
88
81
61
76
64
58
50
55
52
42
54
52
46
35
42
29
39
29
38
40
44
34
19
38
33
31
25
20
18
21
29
19
25
23
12
14
19
17
28
27
24
20
20
13
25
20
19
16
18
26
19
13
24
16
24
25
14
13
13
12
15
13
19
15
8
15
15
12
23
21
12
7
11
18
13
13
16
7
15
14
8
12
13
8
8
10
13
8
7
9
9
8
12
9
11
8
7
12
4
6
7
9
13
9
15
6
11
11
11
13
11
9
9
12
9
6
13
9
14
11
6
9
7
13
10
9
5
9
11
7
10
11
9
6
15
17
10
6
13
7
10
9
9
10
12
11
13
7
6
6
7
4
6
13
7
10
9
6
6
10
9
5
7
5
7
8
9
9
12
12
10
10
9
10
15
6
12
5
11
7
16
10
4
10
7
7
9
15
7
11
7
11
8
5
7
8
7
9
3
15
5
11
7
7
3
9
13
11
7
10
8
5
4
8
6
11
9
6
8
11
7
10
9
13
9
6
5
7
6
10
8
9
12
11
10
12
12
7
8
6
14
12
11
8
8
11
13
13
17
7
6
10
7
14
8
10
5
9
8
8
4
8
7
5
5
3
6
3
5
8
9
6
8
5
10
6
7
7
3
9
6
4
2
9
3
4
8
5
5
6
4
10
5
3
7
4
5
5
10
10
8
7
2
8
6
6
4
6
4
3
3
2
6
6
7
10
6
3
5
4
4
1
4
5
7
4
8
7
7
3
6
2
10
9
3
6
8
5
7
5
4
6
3
4
6
7
7
5
5
2
5
5
6
11
7
5
9
9
6
9
6
10
10
9
4
2
7
5
10
6
4
6
9
6
2
8
7
9
4
4
5
5
8
5
8
3
4
8
4
7
6
2
7
8
2
7
5
4
11
3
6
4
9
4
8
2
8
3
6
6
11
4
7
13
5
3
3
11
8
7
8
3
5
9
4
7
3
5
4
8
6
5
7
5
11
5
3
4
8
4
9
9
6
3
2
3
5
8
3
10
8
9
7
6
4
13
11
5
5
4
11
6
7
6
4
4
11
9
9
3
9
4
9
5
7
6
7
11
9
5
12
5
9
4
2
10
3
2
4
5
2
6
6
11
6
8
11
11
6
8
6
4
6
7
8
8
9
6
3
12
11
8
9
6
8
7
7
6
11
6
8
6
11
10
6
10
9
10
11
9
8
6
9
4
8
4
10
11
11
12
6
9
5
5
10
8
13
7
11
10
8
9
9
6
10
8
12
11
11
7
12
12
11
11
12
15
7
17
13
10
10
13
13
10
13
14
11
11
6
15
9
11
14
15
18
8
15
17
8
17
17
11
17
9
12
22
16
14
21
16
11
16
15
13
8
15
14
22
11
12
20
5
22
25
23
10
14
21
17
15
22
28
20
28
25
18
20
31
42
51
26
37
40
48
52
36
40
49
39
49
32
37
30
34
32
32
35
31
26
25
26
19
26
25
24
22
36
23
22
32
26
28
29
32
34
35
35
34
43
27
29
41
35
31
40
41
31
32
49
47
31
36
54
52
52
63
48
54
57
62
65
60
60
66
85
78
95
132
142
177
226
289
322
422
513
612
745
879
1061
1194
1360
1508
1676
1838
2078
2111
2267
2390
2413
2535
2449
2355
2284
2145
1941
1843
1689
1420
1166
1045
888
742
651
481
369
313
227
170
145
116
97
76
57
64
54
53
47
52
20
32
37
32
24
22
22
21
25
17
24
30
20
23
11
19
21
15
12
17
22
20
21
24
23
20
22
19
22
23
23
14
20
12
10
26
19
23
15
19
21
25
19
24
22
20
31
33
30
23
36
33
43
37
37
50
66
77
88
84
105
146
153
215
239
299
324
384
479
518
594
676
730
844
776
856
872
900
931
880
868
810
803
748
687
596
547
469
411
398
365
328
271
260
224
196
173
163
144
143
130
109
108
81
73
54
60
46
46
48
21
20
14
11
14
6
4
7
7
2
1
3
8
6
7
7
1
5
1
1
4
8
2
0
3
0
2
8
3
3
3
6
5
5
3
3
3
1
1
4
2
1
1
2
3
5
4
4
2
5
7
6
3
10
7
7
12
16
18
22
25
19
34
36
39
45
59
57
72
79
71
76
78
103
78
105
89
82
67
79
67
76
58
68
64
59
30
48
38
32
34
33
29
29
28
32
21
22
21
10
18
13
16
13
12
8
14
3
4
4
5
4
3
3
3
4
2
1
3
0
1
2
2
0
0
1
0
0
1
0
1
0
0
1
0
0
0
File: C:\xrfdaten\2014KW30\chris\std30kev\std10042.spe
Start: 22.07.2014 10:12:46
Stop: 22.07.2014 10:14:27
BESSY: 298.535295 298.6700068
IONI: 1.96394E-11 1.968772E-11
ScanMotor1 exp_x: 48.05 48.05
ScanMotor2 exp_y: 83.45 83.5
Live: 69022
Run: 100000
Dead: 31
ICR: 103190
OCR: 0
Accumulated BESSY: 59706.4472486
Accumulated Ioni: 3.93182882E-09
N Accumulation: 200
Normfactor:  7.3696668e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Mo
 
exp_x 72.7203979492188 72.7204
exp_y 92.4657974243164 92.4658
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 860.6 860.6000079968
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 35 35.0001
dcm_energy 23 -184976.74536672
S3_horSize 0.05 4.99999999999997E-02
S3_verSize 0.05 5.00000000000002E-02
$PEAKS:   
      1
  42   1     17.443   5843868.      2418.     107.99
