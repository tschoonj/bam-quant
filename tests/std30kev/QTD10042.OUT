     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:28:50  บ   
     บ  Spectrum: QTD10042.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 130 -1800        4 iterations done    บ   
     บ  ChiSquare =   169.1   last change =   .01%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -59.4 ๑   100.0   บ     -59.4 ๑      .6   บ   
     บ  GAIN (eV/ch)  บ     20.242 ๑   2.000   บ    20.242 ๑    .001   บ   
     บ det NOISE (eV) บ      -67.0 ๑    40.0   บ     -67.0 ๑    -6.8   บ   
     บ  FANO factor   บ       .154 ๑    .050   บ      .154 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Mo-Ka                           5843868. ๑   2418.           บ   
     บ    KA1        17.479    .65574     3832066. ๑   1586.    125.7  บ   
     บ              866.418    230.48         900.           9.83E-01  บ   
     บ    KA2        17.374    .34426     2011802. ๑    833.     95.3  บ   
     บ              861.231    229.73         921.           9.83E-01  บ   
     บ                                                                 บ   
     บ  2 Mo-Kb                           1227820. ๑   1109.           บ   
     บ    KB1        19.602    .86946     1067540. ๑    964.     91.5  บ   
     บ              971.297    245.19         587.           9.58E-01  บ   
     บ    KB2        19.965    .13054      160280. ๑    147.     19.8  บ   
     บ              989.230    247.62         532.           9.52E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           48290. ๑    222.           บ   
     บ    L1O3       34.853    .45149       21802. ๑    103.     14.0  บ   
     บ             1724.715    332.24         480.           4.90E-01  บ   
     บ    L1M5       34.958    .43000       20765. ๑     98.      9.6  บ   
     บ             1729.902    332.76         485.           4.87E-01  บ   
     บ    L1M4       34.748    .11851        5723. ๑     34.     16.8  บ   
     บ             1719.528    331.72         474.           4.92E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
